src/modules/dashboard/components/Dialogs/RaiseObjectionDialog/RaiseObjectionDialogForm.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { FormattedMessage, defineMessages } from 'react-intl';
+import { Editor } from '@tiptap/react';
 import { FormikProps } from 'formik';
 import Decimal from 'decimal.js';
 
 import Button from '~core/Button';
 import DialogSection from '~core/Dialog/DialogSection';
 import ExternalLink from '~core/ExternalLink';
-import { Annotations } from '~core/Fields';
+import { InputLabel } from '~core/Fields';
+import RichTextEditor from '~core/RichTextEditor/RichTextEditor';
```

Comment:
Can we add a `index.ts` to this folder? :)

Reply:
Done 👍